src/org/zanata/jenkins/Notifier.groovy

```diff
@@ -22,6 +30,67 @@ class Notifier implements Serializable {
     }
   }
 
+  // build: Build object. Use currentBuild
+  // context: the current progress of the build, like UNIT, WILDFLY8, JBOSSEAP
+  // message: The message to be print. Default is ''
+  // result: The result to be set
+  void updateBuildStatus(def build, String context, String message = '', String result = null ){
+    if ( result != null ){
+      build.result = result
+    }
+    switch(context){
+      case CONTEXT_STARTED:
+        started();
```

Comment:
But come to think of it, STARTED does not require to run testResults. So you are right. :-)